src/main/java/com/fasterxml/jackson/datatype/guava/GuavaDeserializers.java

```diff
@@ -18,6 +12,11 @@
 import com.fasterxml.jackson.datatype.guava.deser.multimap.list.LinkedListMultimapDeserializer;
 import com.fasterxml.jackson.datatype.guava.deser.multimap.set.HashMultimapDeserializer;
 import com.fasterxml.jackson.datatype.guava.deser.multimap.set.LinkedHashMultimapDeserializer;
+import com.google.common.base.Optional;
+import com.google.common.collect.*;
+import com.google.common.hash.HashCode;
+import com.google.common.net.HostAndPort;
+import com.google.common.net.InternetDomainName;
```

Comment:
It would be nice not to rearrange import statements, and other cosmetic things, since that adds fluff to diffs.